src/components/Settings/UsageReportSettings.tsx

```diff
@@ -0,0 +1,40 @@
+import { Flex, Text, Checkbox, Link } from '@radix-ui/themes'
+import { Controller, useFormContext } from 'react-hook-form'
+import { SettingsSection } from './SettingsSection'
+import { AppSettings } from '@/types/settings'
+
+export const UsageReportSettings = () => {
+  const { control, register } = useFormContext<AppSettings>()
+
+  const handleLinkClick = () => {
+    window.studio.browser.openExternalLink(
+      'https://github.com/grafana/k6-studio/blob/main/README.md#usage-report'
+    )
+  }
+
+  return (
+    <SettingsSection title="Usage Report">
+      <Flex gap="2">
+        <Controller
+          control={control}
+          name="usageReport.enabled"
+          render={({ field }) => (
+            <Text size="2" as="label">
+              <Checkbox
+                {...register('usageReport.enabled')}
+                defaultChecked
+                checked={field.value}
+                onCheckedChange={field.onChange}
+              />{' '}
+              Please send my anonymous usage data to Grafana to aid in
+              development of k6 Studio.{' '}
```

Comment:
I'd avoid using "please" (as mentioned in [Grafana's UX writing guide](https://grafana.com/docs/writers-toolkit/write/style-guide/ux-writing/#use-a-voice-thats-friendly-and-not-too-formal))